obstacle_rich_env/configs/common_configs.py

```diff
@@ -7,7 +7,8 @@
     max_episode_steps=100,
     observation_flatten=False,
     # obs_key_to_return=["state", "custom_state", "goal_robot_diff", "barriers", "min_barrier"],        # provide a list of keys to include in the observation
-    obs_key_to_return=["state", "custom_state", "goal_robot_diff", "barriers"],          # provide a list of keys to include in the observation
+    # obs_key_to_return=["state", "custom_state", "goal_robot_diff", "barriers"],          # provide a list of keys to include in the observation
+    obs_key_to_return=["state", "custom_state", "goal_robot_diff", "barriers", 'obs_lidar'],
```

Comment:
I suggest changing the key name to obstacle_lidar

Reply:
Variable names have been changed as you suggested.